data/monster/monsters/apprentice_sheng.xml

```diff
@@ -30,7 +30,7 @@
 		<attack name="energyfield" interval="1000" chance="8" range="7" radius="1" target="1">
 			<attribute key="shootEffect" value="energy" />
 		</attack>
-		<attack name="effect" interval="10500" chance="100" radius="3" target="0">
+		<attack name="effect" interval="10500" radius="3" target="0">
```

Comment:
Is `effect` a valid attack name?

Reply:
apparently he is supposed to do that redshimmer area when trying to summon the hyenas

Reply:
then it will need to be moved to Lua, same as werewolf summoning with effects

but I'll open an issue suggesting addition of areaEffect/radius/target to the summon block, it will be very useful as lots of monsters use such setting